src/array_serde.rs

```diff
@@ -91,6 +91,17 @@ impl<S, Di> ArrayVisitor<S, Di> {
     pub fn new() -> Self {
         ArrayVisitor { _marker_a: PhantomData, _marker_b: PhantomData, }
     }
+
+    pub fn verify_version<E>(v: u8) -> Result<(), E>
```

Comment:
Thanks. This is best as a free function actually, since it doesn't use the S, Di type parameters.